### ISSUE TITLE
Cannot extract more than one instance of a front matter key

### ISSUE THREAD

**Issue by jasedit** [fletcher/MultiMarkdown-5#41]
When using MultiMarkdown with LaTeX output, it is common to have multiple front matter keys for inputting files, as shown in the usage guide:

```
latex input: mmd-memoir-header
Title: MultiMarkdown Example
Base Header Level: 2
latex mode: memoir
latex input: mmd-memoir-begin-doc
latex footer: mmd-memoir-footer
```

Attempting to extract the `latex input` key from the command line will only print the first value, with no method for accessing the subsequent `latex input` fields.

**Comment by fletcher** [comment]
That's correct.  In general, there is only one allowed instance of each key.  The latex input is the one exception, as it is handled very differently than the others.
